### PR TITLE
docs: grafana greptimedb docs

### PR DESCRIPTION
**Pull request by sunchanglong**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

## Summary by CodeRabbit

- **Documentation**
  - Updated installation instructions and connection settings for GreptimeDB Data Source Plugin in Grafana.
  - Added guidance on creating dashboards with GreptimeDB and using PromQL for panels.
  - Updated URLs and provided Docker setup instructions for integrating GreptimeDB, Prometheus, and Grafana.
  - Enhanced Chinese documentation with new installation and configuration steps for GreptimeDB in Grafana.

### CODE REVIEW

**Comment by sunchanglong** [comment]
先帮 review 下中文版，然后我再弄下英文版

**Comment by sunchanglong** [comment]
英文文档已加

File: docs/nightly/en/db-cloud-shared/clients/grafana-integration.md [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+docker run -d -p 3000:3000 --name=grafana --rm \
+  -e "GF_INSTALL_PLUGINS=https://github.com/GreptimeTeam/greptimedb-grafana-datasource/releases/latest/download/info8fcc-greptimedb-datasource.zip;info8fcc" \
+  grafana/grafana-oss
+```
+
+### Connection Settings
+
+Click the Add data source button and select GreptimeDB as the type.
+
+- Fill in the `http://<host>:4000` in the GreptimeDB server URL field.
```
Comment (sunng87) [inline docs/nightly/en/db-cloud-shared/clients/grafana-integration.md:43]: Note that on cloud, this is `https://<host>`

File: docs/nightly/en/db-cloud-shared/clients/grafana-integration.md [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
+- Fill in the database name in the Database Name field, default is `public` (not set by default, no need to fill in).
```
Comment (sunng87) [inline docs/nightly/en/db-cloud-shared/clients/grafana-integration.md:44]: Here we can use `<dbname>` placeholder

**Review by nicecui — APPROVED** [review]

**Review by sunng87 — APPROVED** [review]

**Pull request merged**